commands/command_e2e_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestModifyCommandCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	// create new command
+	newCmd := &model.Command{
+		CreatorId: s.th.BasicUser.Id,
+		TeamId:    s.th.BasicTeam.Id,
+		URL:       "http://nowhere.com",
+		Method:    model.COMMAND_METHOD_POST,
+		Trigger:   "trigger",
+	}
+
+	command, _ := s.th.SystemAdminClient.CreateCommand(newCmd)
+
+	s.RunForAllClients("modifyCommandCmdF", func(c client.Client) {
+		printer.Clean()
+		// Reset the cmd and parse to force Flag.Changed to be true.
+		cmd := CommandModifyCmd
+		cmd.ResetFlags()
+		addCommandFieldsFlags(cmd)
+		err := cmd.ParseFlags([]string{
+			command.Id,
+		})
+		s.Require().Nil(err)
+
+		err = modifyCommandCmdF(s.th.SystemAdminClient, cmd, []string{command.Id})
```

Comment:
This doesn't actually modify the command. I suggest modifying the command (maybe change the url) and then checking  that the command was actually modified. 
`RunForAllClients` runs the test function three times, once for each kind of client - System Admin, Local and User, so command should be reset between each test run.

Reply:
Updated to modify the url and then validated.
Also reset between each run

Reply:
@JeremyShih this only resets the command object here. It doesn't reset the command created on the server. After the first client run, the command on the server has the url set to `http://nowhere.com_modified`, and we can't be sure if the test runs for other clients will change it. 
I've added a suggestion for how we can get a unique url for each client's test run.

Reply:
Thanks @ashishbhate!
I fixed some lint issue and it should be good.

The CircleCI Checks failed now and it says TestDeleteUsersCmd did not pass.
But I tried to run `make test-e2e` on local machine and it passed.
Could you help check with it?

Reply:
@JeremyShih can you try merging master in your branch? I think your branch has diverged from master quite a bit. Let me know if you need help sorting out the merge conflict.

Reply:
It's good now after rebasing.